timescaledb/how-to-guides/update-timescaledb/downgrade-timescaledb.md

```diff
@@ -0,0 +1,34 @@
+# Downgrade TimescaleDB versions
+If you upgrade your TimescaleDB version and encounter problems, you can
+immediately roll back your upgrade. To downgrade to your previous version,
+connect to your database and run:
+```sql
+ALTER EXTENSION timescaledb UPDATE TO '<PREVIOUS_VERSION>';
+```
+
+For example:
+```sql
+ALTER EXTENSION timescaledb UPDATE TO '2.5.1';
+```
+
+<highlight type="important"> 
+If you're connecting to your database with `psql`,
+connect with the `-X` flag. This prevents any commands in `.psqlrc` from loading
+another TimescaleDB version. 
+</highlight>
+
+You can verify that the downgrade worked by running `\dx timescaledb` to see the
+version number of the extension.
+
+## Limitations
+Downgrading might not be supported for all versions. Where possible, downgrades
+between patch versions and between consecutive minor versions are supported. For
+example, you can downgrade from TimescaleDB 2.5.2 to 2.5.1, or from 2.5.0 to
+2.4.2. However, some versions might not support downgrading. To check whether
```

Comment:
@mkindahl I checked the release notes for 2.5.2 and 2.5.0, and didn't see any warnings about downgrading. But I want to double-check that these specific versions do support downgrades?

Reply:
We have none yet, but there is bound to be cases in the future where this is not supported.